src/utils/components/Charts/utils/utils.ts

```diff
@@ -1,5 +1,10 @@
 import DurationOption from '@kubevirt-utils/components/DurationOption/DurationOption';
-import { PrometheusResponse, PrometheusValue } from '@openshift-console/dynamic-plugin-sdk';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import {
+  PrometheusResponse,
+  PrometheusResult,
+  PrometheusValue,
+} from '@openshift-console/dynamic-plugin-sdk';
 
 export const SINGLE_VM_DURATION = 'SINGLE_VM_DURATION';
 export const TICKS_COUNT = 100;
```

Comment:
I'd move those constants into a separate constants.ts file ;) WDYT?

Reply:
It's a good idea but it requires me to change the import to many more files and it creates a load on this PR..
so maybe in another PR? :)